include/zephyr/input/input_kbd_matrix.h

```diff
@@ -38,6 +38,33 @@ typedef uint8_t kbd_row_t;
 #define PRIkbdrow "%02x"
 #endif
 
+/** Storage type for actual-key-mask */
```

Comment:
this won't be picked up by Doxygen, as it doesn't sit on top of the define below (you can use `@def` if defining outside).

Is this define meant to be public?

Reply:
no it's meant to stay internal, I dropped to comment, not too sure why I added it in the first place